sample/legacy_with_buckets.go

```diff
@@ -0,0 +1,35 @@
+package sample
+
+import (
+	metrics "github.com/rcrowley/go-metrics"
+)
+
+type LegacyWithBuckets interface {
```

Comment:
this interface is intentionally exposed and to be used by the clients (= `go-service` for now) to retrieve required values.